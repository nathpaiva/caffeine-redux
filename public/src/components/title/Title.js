import React from 'react';

const subTitleCenter = {
  textAlign: 'center',
  fontSize: '20px',
  color: '#414141',
  fontFamily: '"Roboto", sans-serif'
}

const Title = ({ title }) => (
  <h2 style={subTitleCenter}>{title}</h2>
);

export default Title;
