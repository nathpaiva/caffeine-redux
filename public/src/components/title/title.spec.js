'use stict';

import React from 'react';
import renderer from 'react-test-renderer';

import Title from './Title';


describe('#Test Title Component', () => {
  it('#Should Title to mach snapshot', () => {
    const tree = renderer.create(<Title title='Cadastre-se' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
