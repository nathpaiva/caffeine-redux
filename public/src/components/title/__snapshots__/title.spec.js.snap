// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test Title Component #Should Title to mach snapshot 1`] = `
<h2
  style={
    Object {
      "color": "#414141",
      "fontFamily": "\\"Roboto\\", sans-serif",
      "fontSize": "20px",
      "textAlign": "center",
    }
  }
>
  Cadastre-se
</h2>
`;
